examples/units_and_coordinates/AltAz_Coordinate_Transfer_.py

```diff
@@ -14,7 +14,7 @@
 import astropy.units as u
 
 ######################################################################################
-# We use ~astropy.coordinates.SkyCoord~ to define the center of the Sun
+# We use '~astropy.coordinates.SkyCoord' to define the center of the Sun
```

Comment:
this needs to be the ``` ` ``` quote not the `'` quote. :smile:

Reply:
All good, appreciate it, @Cadair